internal/btf/ext_info.go

```diff
@@ -265,92 +375,71 @@ type coreRelo struct {
 	kind     COREKind
 }
 
-type coreRelos []coreRelo
-
-// append two slices of extInfoRelo to each other. The InsnOff of b are adjusted
-// by offset.
-func (r coreRelos) append(other coreRelos, offset uint64) coreRelos {
-	result := make([]coreRelo, 0, len(r)+len(other))
-	result = append(result, r...)
-	for _, relo := range other {
-		relo.insnOff += uint32(offset)
-		result = append(result, relo)
-	}
-	return result
-}
+type CoreRelos []coreRelo
```

Comment:
Not really an issue atm, since the package is still not public, but I've changed it for consistency's sake. It was also poking my OCD. :slightly_smiling_face: